src/main/java/org/simplify4u/plugins/keyserver/PGPKeyNotFound.java

```diff
@@ -21,6 +21,7 @@
  * Inform about PGP key not found on server.
  */
 public class PGPKeyNotFound extends IOException {
+    private static final long serialVersionUID = 1L;
```

Comment:
It can be also enabled in sonar rules

Reply:
I don't seem to have permission to change the Sonar quality profile.

Reply:
@slawekjaranowski can you have a look at Sonar. I don't seem to have the right permissions.

Reply:
now you should

Reply:
This rule is already enabled in Sonar.